libs/wingsdk/src/cloud/bucket.md

```diff
@@ -0,0 +1,70 @@
+---
+title: cloud.Bucket 
+id: bucket
+description: A built-in resource for handling object storage in the cloud.
+keywords: [Wing reference, Wing language, language, Wing sdk, Wing programming language, secrets]
+---
+
+The `cloud.Bucket` resource represents a container for storing data in the cloud.
+
+Buckets are a common way to store arbitrary files (images, videos, etc.), but can also be used to store structured data like JSON or CSV files.
+
+Buckets in the cloud use object storage, which is optimized for storing large amounts of data with high availability.
+Unlike other kinds of storage like file storage, data is not stored in a hierarchical structure, but rather as a flat list of objects.
+
+## Usage
+
+### Creating a bucket
```

Comment:
I am wondering if we should use the "define" terminology (i.e. "Defining a bucket").

Reply:
I like the idea